Gems/Atom/Tools/MaterialEditor/Code/Source/Window/MaterialEditorWindow.cpp

```diff
@@ -425,8 +416,8 @@ namespace MaterialEditor
             if (!result)
             {
                 const QString documentPath = GetDocumentPath(documentId);
-                const QString status = QString("Failed to perform Undo in material: %1").arg(documentPath);
-                m_statusBar->setWindowIconText(QString("<font color=\"Red\">%1</font>").arg(status));
+                const QString status = QString("Failed to perform undo on document: %1").arg(documentPath);
```

Comment:
Nitpick: Would be to change this (and the others being modified in this file) to tr while you're here so it can be translated at some point.